### DISCUSSION TITLE
Scrambled Bugs

### DISCUSSION THREAD

**Discussion by mccllc** [alandtse/skyrim_vr_address_library#33] [Ideas]
Several of the mods I enjoy have soft/hard dependencies on this bug fix collection. It would be great see it available in VR:

https://www.nexusmods.com/skyrimspecialedition/mods/43532

**Comment by alandtse** [comment]
VR port was done [here](https://www.nexusmods.com/skyrimspecialedition/mods/107053).

**Reply by mccllc** [reply]
Thank you!
